build/targets/codeanalysis/CodeAnalysis.props

```diff
@@ -5,6 +5,8 @@
     <AnalysisMode>preview</AnalysisMode>
     <WarningLevel>9999</WarningLevel>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
How do you feel about hooking this into `$(IsContinuousIntegration)`, or a new property that defaults to IsContinuousIntegration?

Otherwise, local dev gets hung up on things like incorrect style, unused parameters, etc.

Reply:
I'm fine with that. You want to open a new PR for that, or put it in this one?

Reply:
I pushed df2af35 with the flag